starter-code/index.html

```diff
@@ -3,40 +3,82 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="./css/main.css"/>
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+  <nav class="navBar">
+    <img src="./images/spotify-logo.png">
+    <ul>
+        <li href="https://www.google.com">Premium</li> 
+        <li href="https://www.google.com">Discover</li> 
+        <li href="https://www.google.com">Help</li> 
+        <li href="https://www.google.com">Download</li>
+    </ul>
+  
+
+  </nav>
+ <div class="header">
+
+ <h1> Music for everyone. </h1>
+
+  <h3>Spotify is now free on mobile, tablet and computer. <br>
+      Listen to the right music, wherever you are.</h3>
+ </div>
+
+
+ <section>
+  <h3> <strong> What’s on Spotify? </strong>  </h3>
+  <div class="songs">
+      <img src="./images/music-icon.png" height="140" width="140">
```

Comment:
Actually it's not a good practice to put inline style like this, and as you are applying the same style on your three elements, you should use class and put it in your css file.

Also small trick when it comes to resize image, use a width like you did and use auto for you height.

```css

#green-container-img img {
  width: 60%;
  height: auto;
  float: right;
}

```

This is the code I use for this exercise, like this your image will keep the same proportions.